examples/bin/start-druid-main.py

```diff
@@ -436,43 +428,39 @@ def check_memory_constraint(total_memory, services):
     return int(total_memory * 0.8)
 
 
-def build_mm_task_java_opts_array(memory_type):
-    task_memory = '-D{0}=['.format(TASK_JAVA_OPTS_PROPERTY)
-
-    mem_array = TASK_MEM_MAP.get(memory_type)
+def build_mm_task_java_opts_array(task_memory):
+    tasks_memory = '-D{0}=['.format(TASK_JAVA_OPTS_PROPERTY)
 
+    mem_array = ["-Xms{0}m".format(task_memory), "-Xmx{0}m".format(task_memory), "-XX:MaxDirectMemorySize={0}m".format(task_memory)]
     java_opts_list = TASK_JAVA_OPTS_ARRAY + mem_array
-
     for item in java_opts_list:
-        task_memory += '\"{0}\",'.format(item)
+        tasks_memory += '\"{0}\",'.format(item)
 
-    task_memory = task_memory[:-1]
-    task_memory += ']'
-    return task_memory
+    tasks_memory = tasks_memory[:-1]
+    tasks_memory += ']'
```

Comment:
style nit: it might be easier to read if this line is replaced with:
```suggestion
    tasks_memory += '-D{0}=[{1}]'.format(TASK_JAVA_OPTS_PROPERTY, task_java_opts_value)
```
and line 432 can be removed.